applications/RANSApplication/custom_strategies/bossak_scalar_scheme.h

```diff
@@ -0,0 +1,371 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:   Jordi Cotela
+//                  Suneth Warnakulasuriya
+//
+
+#if !defined(KRATOS_BOSSAK_SCALAR_TRANSPORT_SCHEME_H_INCLUDED)
+#define KRATOS_BOSSAK_SCALAR_TRANSPORT_SCHEME_H_INCLUDED
+
+// System includes
+#include <sstream>
+#include <vector>
+
+// External includes
+
+// Project includes
+#include "custom_strategies/relaxed_dof_updater.h"
+#include "includes/define.h"
+#include "includes/model_part.h"
+#include "solving_strategies/schemes/scheme.h"
+#include "utilities/time_discretization.h"
+
+// Application includes
+#include "custom_strategies/steady_scalar_scheme.h"
+
+namespace Kratos
+{
+///@name Kratos Classes
+///@{
+
+/// A scheme for steady and dynamic equations, using Bossak time integration.
+/**
+ * It can be used for either first- or second-order time derivatives. Elements
+ * and conditions must provide a specialization of SchemeExtension via
+ * their data value container, which allows the scheme to operate independently
+ * of the variable arrangements in the element or condition.
+ */
+template <class TSparseSpace, class TDenseSpace>
+class BossakScalarScheme
```

Comment:
I find that the class name is misleading. What I would expect from the current one is a Bossak scheme to integrate in time any scalar variable without any relaxation. What do you think about `BossakRelaxationScalarScheme` or something like that.

Reply:
done :)